content/en/docs/chart_best_practices/templates.md

```diff
@@ -31,62 +31,73 @@ For that reason, _all defined template names should be namespaced._
 Correct:
 
 ```yaml
-{{- define "nginx.fullname" }}
-{{/* ... */}}
-{{ end -}}
+{{- define "nginx.fullname" -}}
+# ...
```

Comment:
It's not totally clear here why you swapped out the Go template comment syntax with YAML comments. The former will be removed from the rendered output, while the latter will not. It also introduces the user to the text/template syntax.

https://golang.org/pkg/text/template/#hdr-Actions

Can you please revert this as well as any other similar changes in this PR?

Reply:
Okay I'll retain template comments instead of YAML comments. I made the switch because I felt it increased readability, partially because syntax highlighting and partially because it felt like it communicated better that it was placeholder content.